docs/cs/BuildTest.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Threading;
+using System.Numerics;
+using DafnyVMC;
+
+class Check {
+    public static void Main(string[] args) {
```

Comment:
The C# back-end does not support adding a Main method from an external file. Why not make this a Dafny file?